services/wallet/activity/activity.go

```diff
@@ -198,6 +199,9 @@ const (
 	//
 	// UNION ALL is used to avoid the overhead of DISTINCT given that we don't expect to have duplicate entries outside
 	// the sender and receiver addresses being in the list which is handled separately
+	//
+	// Only status FailedAS, PendingAS and CompleteAS are returned. FinalizedAS is for bundled transactions
```

Comment:
not sure if I'm mixing concepts, didn't the "Finalized" state mean the block for this transaction is [finalized](https://docs.symbol.dev/concepts/block.html#finalization)? In that case, it wouldn't matter if its a single transaction or a transaction bundle.

Reply:
You are right about removing the bundle reference.